client/src/components/ScrollToTop.tsx

```diff
@@ -0,0 +1,22 @@
+import { Component } from 'react';
+import { withRouter, RouteComponentProps } from "react-router";
+
+const APP_BODY_SELECTOR = '.App__body';
+
+class ScrollToTop extends Component<RouteComponentProps> {
+  componentDidUpdate(prevProps: RouteComponentProps) {
+    if (this.props.location !== prevProps.location) {
+      const appBody = document.querySelector(APP_BODY_SELECTOR);
+      if (!appBody) {
+        throw new Error(`"${APP_BODY_SELECTOR}" does not exist!`);
```

Comment:
Here we're just throwing a more informative error than the `Cannot set property 'scrollTop' of null` error that would've been thrown if the `querySelector` call failed (while also appeasing TypeScript).

Reply:
noice!